frontend/src/views/Simulator/SettingsView.vue

```diff
@@ -65,7 +65,8 @@ async function handleDeleteValidator() {
     notify({
       title: 'OK',
       text: 'Validator deleted successfully',
-      type: 'success'
+      type: 'success',
+      duration: 53000
```

Comment:
@bradleySuira is this on milliseconds? why 53000?

Reply:
fixed, removed the duration